examples/Example04_SYND_Mode/Example04_SYND_Mode.ino

```diff
@@ -51,7 +53,7 @@ void setup() {
   Serial.println("Sensor began.");
 
   // Set measurement mode and change device operating mode to measure.
-  if(myUVSensor.startMeasurement(MEAS_MODE_SYND) == false) {
+  if(myUVSensor.prepareMeasurement(MEAS_MODE_SYND) == false) {
```

Comment:
I think it'd help to add a comment to explain that `MEAS_MODE_SYND` is "SYNchronous start and enD mode", like in the `as7331_meas_mode_t` definition. It'd be easy for a user to miss the difference between it and `MEAS_MODE_SYNS`

Reply:
It's explained in the top comment description.